runtime/trampolines.m

```diff
@@ -80,8 +80,6 @@
 	/* Any changes in this method probably need to be reflected in the static registrar as well */
 	switch (type [0]) {
 		case _C_CLASS:
-			return xamarin_get_handle_for_inativeobject (retval, exception_gchandle);
```

Comment:
Isn't the function expected to return some sort of void pointer? If the function hits this empty case it will never return.

Reply:
it's not an empty case, i.e. there's no `break` or `continue`, so it just share the same code path as `_C_SEL`

Reply:
@wachaudh that is what they mean when they say a switch case will fall-through.